Source/Core/VideoCommon/TextureCacheBase.h

```diff
@@ -199,6 +200,11 @@ class TextureCacheBase
   TexPool::iterator FindMatchingTextureFromPool(const TCacheEntryConfig& config);
   TexAddrCache::iterator GetTexCacheIter(TCacheEntryBase* entry);
 
+  // Return all possible overlapping textures. As addr+size of the textures is not
+  // indexed, this may return false positives.
+  std::pair<TexAddrCache::iterator, TexAddrCache::iterator>
+  FindOverlappingTextures(u32 addr, u32 size_in_bytes);
```

Comment:
Don't worry. I didn't know this either, so it sounds nice. But GCC didn't like it, and now I know why :D So thank you.